tests/test_wave_01.py

```diff
@@ -183,12 +184,17 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
+    janes_data = {
+        "watchlist": [FANTASY_1],
+        "watched": [FANTASY_2, movie_to_watch]
+    }
+    assert updated_data == janes_data
```

Comment:
Only suggestion I have here is to change name to something else like `expected`. Also see my note below about [making copies of the originals](https://github.com/Ada-C22/viewing-party/pull/14/files#r1792645935).